components/Gallery/Gallery.tsx

```diff
@@ -1,22 +1,30 @@
+
+import dynamic from 'next/dynamic';
+import lgZoom from 'lightgallery/plugins/zoom';
+
 import { GalleryItem } from "./GalleryItem";
 import styles from "./Gallery.module.scss";
-import { SRLWrapper } from "simple-react-lightbox";
 import { Section } from "../Section/Section";
 
+import 'lightgallery/css/lightgallery.css';
+import 'lightgallery/css/lg-zoom.css';
+
 interface Props {
-  readonly images: { src: string; title: string }[];
+  readonly images: { src: StaticImageData; title: string }[];
 }
 
+const LightGallery = dynamic(() => import('lightgallery/react'), {
```

Comment:
I am not sure about this change. It's working in nextjs but will it affect performance and seo? 
With this lib gallery is finally working as it should (you can check how zooming, loading was working with old lib) so it might be worth it.